source/scpp/src/scp/LocalNode.h

```diff
@@ -9,6 +9,7 @@
 #include <vector>
 
 #include "scp/SCP.h"
+#include "util/HashOfHash.h"
```

Comment:
Okay so I was looking at this locally. This is what pulls in the hash & secretkey dependency AFAICS.
If you look at our fork of SCP, I actually removed it. I believe it could be removed upstream, but it is used in the implementation (the `.cpp`). 

However, I think those are just lingering import. If you remember, Stellar reduced dependencies by exposing a `sign` method, instead of relying on `SecretKey`. So I believe we should be able to remove all `SecretKey`-related code from our code, but we'll have to remove some (unused) imports.